README.md

```diff
@@ -117,6 +117,7 @@ You can hook into events using the triggered event callback props described here
 |     `renderData`     | Object containing the dynamic render data for your contract. [For more information on using dynamic contracts, check out this documentation.](https://developer.pactsafe.com/docs/how-to-use-smart-contracts-with-the-javascript-library)                                                                                                                                          |                                    object                                    |         If `dynamic` is set to true         |                   undefined                  |
 |  `signerIdSelector`  | The ID of the `<input>` element that will be used to identify the signer.                                                                                                                                                                                                                                                                                                          |                                    string                                    |                     Yes                     |                Required Value                |
 |      `signerId`      | Use this to set the signer id directly. Note that if this value is tied to a state variable updated via user input, you may hit rate limits if it is updated frequently in a short period of time. To avoid hitting a rate limit, it is best to set the value tied to this prop only when the user's input is complete as opposed to changing this value on a per character basis. |                                    string                                    | No, unless `signerIdSelector` is not passed |                   undefined                  |
+|     `snapshotLocation`     | Ironclad Clickwrap Snapshot Location Key, this is found within the Snapshot Location configuration                                                                                                                                                                                                                                                                             |                                    string                                    |                      No                     |                   undefined                  |
```

Comment:
The description follows the structure of `groupKey`. I would suggest a different sentence structure to avoid a comma splice and be a little more descriptive.

Something like:
> Snapshot Location key, which is found within the Snapshot Location configuration. Location must be published to tie visual evidence to tracked acceptances.

(I copied some of that language from our [Support documentation](https://support.ironcladapp.com/hc/en-us/articles/12449273874327-Manage-Snapshot-Locations#AddLocationToEvent)).